index.js

```diff
@@ -458,7 +539,7 @@
   def('compose',
       {},
       [$.Function, $.Function, a, c],
-      function(f, g, x) { return f(g(x)); });
+      compose3);
```

Comment:
Yep! In cases where we already mention the similar function in the description this isn't necessary, though we should ensure that each reference is a link to the relevant section of the documentation.